site/content/docs/user/quick-start.md

```diff
@@ -28,7 +28,7 @@ shown [here](https://golang.org/doc/code.html#GOPATH) if you encounter the error
 `kind: command not found` after installation.
 
 **NOTE**: `go get` should not be run from a Go [modules] enabled project directory,
-as go get inside a modules enabled project updates dependencies / behaves differently. Try for example `cd $HOME` first.
+as go get inside a module enabled project updates dependencies / behaves differently. Try for example `cd $HOME` first.
```

Comment:
this one was actually correct, AIUI, we're talking about the collective enablement of "[Go] modules", not about a _module_.

Reply:
see e.g. https://www.jetbrains.com/help/go/create-a-project-with-go-modules-integration.html
> A new Go modules project already has Go modules enabled. If you pulled your Go modules project from Github, you need to enable Go modules manually.

Reply:
Thanks, I see.